core/tee/tee_rpmb_fs.c

```diff
@@ -26,21 +26,24 @@
  */
 
 #include <assert.h>
-#include <kernel/tee_common.h>
 #include <kernel/mutex.h>
 #include <kernel/panic.h>
+#include <kernel/tee_common.h>
 #include <kernel/tee_common_otp.h>
+#include <kernel/tee_misc.h>
 #include <kernel/thread.h>
 #include <mm/core_memprot.h>
 #include <mm/tee_mm.h>
 #include <optee_msg_supplicant.h>
 #include <stdlib.h>
-#include <string.h>
 #include <string_ext.h>
+#include <string.h>
```

Comment:
ok, thanks for the info. I tried it and it gives strange sorting results (at least not want I would expect) when using non alphanum characters. 
```
foo/my_ack.h
foo/my/ack.h
foo/mystery.h
foo/my_string.h
foo/my/string.h
```
nevertheless less, that's quite minor stuff.